pom.xml

```diff
@@ -0,0 +1,290 @@
+        </dependency>
+
+        <!-- Tests -->
+        <dependency>
+            <groupId>io.gravitee.apim.gateway</groupId>
+            <artifactId>gravitee-apim-gateway-tests-sdk</artifactId>
+            <version>${gravitee-apim.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>4.0.0</version>
+            <scope>test</scope>
+        </dependency>
+
+<!--        <dependency>-->
+<!--            <groupId>io.gravitee.gateway.reactive</groupId>-->
+<!--            <artifactId>gravitee-gateway-reactive-tests-sdk</artifactId>-->
+<!--            <version>${gravitee.version}</version>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.13.4.2</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <configuration>
+                        <rules>
+                            <requireReleaseDeps>
+                                <message>No Snapshots Allowed!</message>
+                                <excludes>
+                                    <!-- Workaround to break cyclic dependencies in APIM between plugins and core APIM -->
+                                    <exclude>io.gravitee.*:*:*:*:test</exclude>
+                                    <exclude>io.gravitee.apim.*:*:*:*:provided</exclude>
+                                </excludes>
+                            </requireReleaseDeps>
+                            <requireReleaseVersion>
+                                <message>No Snapshots Allowed!</message>
+                            </requireReleaseVersion>
+                        </rules>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <version>${license-maven-plugin.version}</version>
+                <configuration>
+                    <excludes>
+                        <exclude>*/**</exclude>
+                    </excludes>
```

Comment:
for OSS policies, keep it active. 
Can you compare with a recently created OSS policy that this configuration and the others are good ?